src/authentication/logging/LogConfiguration.js

```diff
@@ -99,6 +121,19 @@ class LogConfiguration {
     }
 
     getDefaultLoggingProperties(warningMessage) {
+
+        if(typeof (this.externalLogger) === "boolean" && this.externalLogger === true){
+            this.externalLogger = true;
+        } else {
+            this.externalLogger = false;
+        }
+
+        if(typeof (this.externalLoggerObj) === "Object" || !(this.externalLoggerObj instanceof ExternalLoggerWrapper)
+            && this.externalLoggerObj === undefined || !(this.externalLoggerObj.isLoggerEmpty())){
```

Comment:
Check this condition properly.

From what I see, you are writing `if(typeof (this.externalLoggerObj) === "Object" && this.externalLoggerObj === undefined)` which is impossible to be true.